articles/azure-maps/web-sdk-best-practices.md

```diff
@@ -20,13 +20,13 @@ Generally, when looking to improve performance of the map, look for ways to redu
 
 ## Security best practices
 
-For security best practices, see [Authentication and authorization best practices](authentication-best-practices.md).
+For more information on security best practices, see [Authentication and authorization best practices](authentication-best-practices.md).
 
 ### Use the latest versions of Azure Maps
 
-The Azure Maps SDKs go through regular security testing along with any external dependency libraries that may be used by the SDKs. Any known security issue is fixed in a timely manner and released to production. If your application points to the latest major version of the hosted version of the Azure Maps Web SDK, it will automatically receive all minor version updates that will include security related fixes.
+The Azure Maps SDKs go through regular security testing along with any external dependency libraries used by the SDKs. Any known security issue is fixed in a timely manner and released to production. If your application points to the latest major version of the hosted version of the Azure Maps Web SDK, it automatically receives all minor version updates that include security related fixes.
 
-If self-hosting the Azure Maps Web SDK via the npm module, be sure to use the caret (^) symbol to in combination with the Azure Maps npm package version number in your `package.json` file so that it will always point to the latest minor version.
+If self-hosting the Azure Maps Web SDK via the npm module, be sure to use the caret (^) symbol to in combination with the Azure Maps npm package version number in your `package.json` file so that it points to the latest minor version.
 
 ```json
 "dependencies": {
```

Comment:
please also update the latest npm version to "^2.2.6", or is there any other presentation that can guide user to use latest version instead we hardcode a certain version here? Then we don't need to update here everytime we had new web sdk version release.
The latest npm package of Azure Maps Control please refer to: https://www.npmjs.com/package/azure-maps-control?activeTab=versions